### ISSUE TITLE
Is the loss of the controller right?

### ISSUE THREAD

**Issue by aandyjeon** [microsoft/nni#2765]
Hi,

Thank you for code you shared!
I found the loss of controller is little bit different from what I understood.
Can I ask if the code of following code is right?

https://github.com/microsoft/nni/blob/master/src/sdk/pynni/nni/nas/pytorch/enas/trainer.py#169
loss = self.mutator.sample_log_prob * (reward - self.baseline)

I think that because the loss decreases during training, the reward also decreases in this code. I believe that the reward should increase.

Thanks in advance.
Myeongho Jeon

**Comment by ultmaster** [comment]
Hi, thanks for raising this issue. `loss` is actually `reward` here. We follow the name convention from melodyguan/enas.

https://github.com/melodyguan/enas/blob/4bcfd73b524627ea96574e5fed33da74bc7855d5/src/cifar10/general_controller.py#L290

**Comment by scarlett2018** [comment]
Closing the issue as the question has been answered. @aandyjeon - thanks for contact and raise question, feel free to reopen the issue if you have further question to discuss on this topic. Stay tune with NNI =).
